metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ministryofjustice/opg-repository-reporting/main/schema/v0.0.1.json",
+  "owners": ["vega", "sirius-supervision"]
```

Comment:
should this have a sirius dependency?

Reply:
I think it's the other way around: Sirius depends on Membrane but Membrane could be stood up alone and/or used for another service (God forbid)